web/src/app/monitors/monitors-list/monitors-list.component.ts

```diff
@@ -42,21 +39,18 @@ export class MonitorsListComponent implements OnInit {
       .subscribe((project: ProjectModel) => this.monitors = project.monitors ? project.monitors : []);
   }
 
-  /**
-   * Lifecycle destroy method
-   */
-  ngDestroy(): void {
-    this.projectSubscription.unsubscribe();
-  }
-
   /**
    * This method is used to delete the monitor from list
    *
    * @param id the id of monitor which needs to be deleted
    */
   deleteMonitor(id: string): void {
     this.monitors = this.monitors.filter((monitor: MonitorModel) => monitor.uid !== id);
-    this.monitorService.saveMonitor(id, this.monitors);
+    this.monitorService.saveMonitor(this.projectUid, this.monitors);
```

Comment:
the `id` should probably be `uid` - to be consistent with everywhere else

Reply:
Its actually uid but here the argument name is id. I have changed it to uid

Reply:
ok thanks 👍